lib/commands/init.js

```diff
@@ -3,6 +3,7 @@
 const fs = require('fs')
 const cp = require('cp-file').sync
 const chalk = require('chalk')
+const inquirer = require('inquirer')
```

Comment:
I think it is better to use [enquirer](https://github.com/enquirer/enquirer) which is a lightweight alternative with a similar API.

Reply:
Let me see

Reply:
Done, But there was one problem.

Mac OS and CentOS symbols are inconsistent, causing CI to not pass. https://travis-ci.org/github/docsifyjs/docsify-cli/jobs/719497023

![image](https://user-images.githubusercontent.com/33931153/90726217-eed3a700-e2f3-11ea-8b79-759ad0ca59e5.png)
![image](https://user-images.githubusercontent.com/33931153/90726246-fa26d280-e2f3-11ea-9510-ecbd4f13a2b2.png)

@jamesgeorge007 @anikethsaha enquirer doesn't support defining symbols yet, what should we do?

![image](https://user-images.githubusercontent.com/33931153/90726418-42de8b80-e2f4-11ea-88f8-7f3ec66decbb.png)

Reply:
Done.